src/bin/vhost_user_net.rs

```diff
@@ -0,0 +1,610 @@
+// Copyright 2019 Intel Corporation. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+//
+// Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+//
+// Portions Copyright 2017 The Chromium OS Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the THIRD-PARTY file.
+
```

Comment:
Since you're going to have to rebase: There should only be one SPDX identifier and in that case it should be `Apache-2.0 AND BSD-3-Clause`

Reply:
Is it the output that you expect?
```rust
// Copyright 2019 Intel Corporation. All Rights Reserved.
//
// Portions Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
//
// Portions Copyright 2017 The Chromium OS Authors. All rights reserved.
//
// SPDX-License-Identifier: Apache-2.0 AND BSD-3-Clause
```

Or did you suggest to keep only the Intel copyright?

Reply:
No, we must keep all copyrights, but have only one identifier.
The above change is almost there: `SPDX-License-Identifier: (Apache-2.0 AND BSD-3-Clause)` ...(missing parenthesis, sorry for the nit).